app/src/main/java/com/github/sdpteam15/polyevents/view/fragments/MapsFragment.kt

```diff
@@ -52,18 +52,18 @@ class MapsFragment : Fragment(), OnMapReadyCallback, OnPolylineClickListener,
     var timerHeatmap: Timer? = null
 
     override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
+            inflater: LayoutInflater,
+            container: ViewGroup?,
+            savedInstanceState: Bundle?
```

Comment:
not sure why there are these double tabulations all over your code, does ctrl+alt+L fix it ?

Reply:
Ctrl + alt+L create them :)

Reply:
It's strange, when I took your branch it deleted them by reformating code, maybe you changed some settings in android studio?